src/wasm/wasm-validator.cpp

```diff
@@ -337,6 +342,82 @@ void FunctionValidator::noteLabelName(Name name) {
     "names in Binaryen IR must be unique - IR generators must ensure that");
 }
 
+void FunctionValidator::visitLoop(Loop* curr) {
```

Comment:
I assume this was just moved around?

Reply:
Yes. I don't remember why it ended up there... Put it back in the original place.